### PR TITLE
[DONE] Create account

### PR DESCRIPTION
**Pull request by Tonirel**
We just need to add what happens when you press the buttons "Create account" and "Cancel" (there are "todo" comments inside the code) which will happen after the other functionalities are finished + change the white background which will happen after the UI design is finished.

### CODE REVIEW

**Comment by andreeasechelaru** [comment]
> We just need to add what happens when you press the buttons "Create account" and "Cancel" (there are "todo" comments inside the code) which will happen after the other functionalities are finished + change the white background which will happen after the UI design is finished.

File: src/app/create-account/create-account.component.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+  }
+
+
+  ngOnInit(): void {
+  }
+
+  onCancel() {
+    //todo: navigate to the previous page
+    console.log("on cancel")
```
Comment (andreeasechelaru) [inline src/app/create-account/create-account.component.ts:65]: in ce cazuri se apeleaza functia asta?

**Review by adipopbv — APPROVED** [review]
Good shit

File: src/app/create-account/create-account.component.html [hunk 2]
```diff
@@ -178,14 +176,59 @@ <h1> Register </h1>
...
             </button>
           </div>
         </div>
-      </div>
-    </div>
+    </form>
   </div>
 </div>
+-->
+<!--Attempt 3 (I promise it's the final!)-->
```
Comment (CristianRotar) [inline src/app/create-account/create-account.component.html]: Attempt 3 looks good, so you can delete the first 2 attempts

**Review by andreeasechelaru — APPROVED** [review]

File: src/app/create-account/create-account.component.html (hunk 2 above)
Comment (Tonirel) [reply src/app/create-account/create-account.component.html]: I will!

File: src/app/create-account/create-account.component.ts (hunk 1 above)
Comment (Tonirel) [reply src/app/create-account/create-account.component.ts:65]: Ai dreptate, pentru ca nu avem buton de cancel, functia nu se apeleaza niciodata. O comentez, in caz ca avem nevoie de ea pe viitor

**Review by adipopbv — APPROVED** [review]

**Pull request merged**